alibi/explainers/anchor_image_sampler.py

```diff
@@ -0,0 +1,260 @@
+import numpy as np
+import copy
+
+from typing import Tuple, Callable, List, Union
+
+from .anchor_image_utils import scale_image
+
+
+class AnchorImageSampler:
+    def __init__(
+        self,
+        # TODO: Should we call `predictor`, `prediction_fn` instead?
```

Comment:
The convention is to call everything `predictor` regardless if it's a white-box model or just a black-box prediction function, partially because some methods accept both types.

Reply:
Makes sense. I'll remove the TODO comment. :+1: